modules/environment/aws/code-services/main.tf

```diff
@@ -202,3 +202,47 @@ resource "aws_sqs_queue_policy" "code_services_queue_policy" {
 }
 POLICY
 }
+
+resource "aws_iam_role" "sqs_role" {
+  count  = var.enable_aws_code_services ? 1 : 0
+  name = "${var.cluster}_sqs_role"
+
+  assume_role_policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "sqs:*",
+                "sts:*"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
```

Comment:
assume role policy shouldn't  have a resource.  also, the action should only be `sts:AssumeRole`.  also, needs a `Principal`